css-highlight-api-1/Overview.bs

```diff
@@ -579,6 +582,101 @@ Range Updating and Invalidation</h3>
+
+		<xmp highlight=html>
+			<style>
+				:root::highlight(foo) {
+					background-color:yellow;
+				}
+				:root::highlight(bar) {
+					color:red;
+				}
+			</style>
+			<body>abc
+			<script>
+				document.addEventListener('click', (event) => {
+					const mouseX = event.clientX;
+					const mouseY = event.clientY;
+					console.log(CSS.highlights.highlightsFromPoint(mouseX, mouseY));
+				});
+
+				let textNode = document.body.firstChild;
+				let r1 = new Range();
+				r1.setStart(textNode, 0);
+				r1.setEnd(textNode, 2);
+				let r2 = new Range();
+				r2.setStart(textNode, 1);
+				r2.setEnd(textNode, 2);
+
+				let h1 = new Highlight(r1);
+				let h2 = new Highlight(r2);
+				h1.priority = 1;
+				h2.priority = 2;
+
+				CSS.highlights.set("foo", h1);
+				CSS.highlights.set("bar", h2);
+			</script>
+		</xmp>
+
+		The code above will display the following styled text, note that "b" is affected by both [=highlight|highlights=]
+		<var>h1</var> and <var>h2</var>, whereas "a" is only affected by <var>h1</var>:
+
+		<div class=sample-out style="color:black">
+			<span style="background:yellow;">a</span><span style="background:yellow;color:red;">b</span><span>c</span>
+		</div>
+
+		In this example there's an [=event listener=] set on click [=event|events=] that logs the [=custom highlights=]
+		present at the point where the click was made.
+		The following [=sequence|sequences=] are some examples of what will be printed to console after a click:
+			* <code>[ <var>h1</var> ]</code>, if the user clicks on character "a".
+			* <code>[ <var>h2</var>, <var>h1</var> ]</code>, if the user clicks on character "b",
+				as <var>h2</var> has higher priority than <var>h1</var>.
+			* <code>[]</code>, if the user clicks on character "c".
+	</div>
+
+The method {{highlightsFromPoint}} is defined as part of the {{HighlightRegistry}} interface as follows:
+
+<pre class=idl>
+partial interface HighlightRegistry {
+	sequence&lt;Highlight> highlightsFromPoint(float x, float y, optional <span>HighlightsFromPointOptions</span> options = {});
+};
+
+dictionary <dfn dictionary>HighlightsFromPointOptions</dfn> {
+	sequence&lt;ShadowRoot> <dfn dict-member for="HighlightsFromPointOptions">shadowRoots</dfn> = [];
+};
+</pre>
+
+The <dfn method for="HighlightRegistry">highlightsFromPoint(<var>x</var>, <var>y</var>, <var>options</var>)</dfn>
+method must return the result of running these steps:
+
+1. If any of the following are true, return the empty [=sequence=]:
+	* <var>x</var> is negative
+	* <var>y</var> is negative
+	* <var>x</var> is greater than the <a>viewport</a> width excluding the size of a rendered scroll bar (if any)
+	* <var>y</var> is greater than the <a>viewport</a> height excluding the size of a rendered scroll bar (if any)
+1. Otherwise, return a [=sequence=] of [=custom highlights=] given by ordering the highlights contained in this {{HighlightRegistry}} by [=priority=],
```

Comment:
```suggestion
1. Otherwise, return a [=sequence=] of [=custom highlights=] given by ordering the highlights contained in this {{HighlightRegistry}} in descending order of [=priority=],
```